airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_poetry/templates/README.md.j2

```diff
@@ -0,0 +1,91 @@
+# {{ connector.name.title()}} source connector
```

Comment:
should we move this doc to a centralized file since the instructions should be the same for most (all?) API source connectors in the repo?

We could instead have a single markdown file in our docs and just point to it from the connector's readme

Reply:
I was thinking about a simpler template that just has a link to the actual doc, but a symlink is a good idea